hack/must-gather.sh

```diff
@@ -43,6 +43,21 @@ if [[ "$ocp_cluster" ]]; then
     $K get clusterversion/version -oyaml > $ARTIFACT_DIR/openshift_version.yaml
 fi
 
+echo
+echo "#"
+echo "# OpenShift HyperConverged Resources"
+echo "#"
+echo
+
+HYPERCONVERGED_RESOURCE=$($K get hco -oname --ignore-not-found)
+
+if [[ "$HYPERCONVERGED_RESOURCE" ]]; then
+    echo "Get HyperConverged YAML"
+    $K get hyperconverged/hyperconverged -oyaml > $ARTIFACT_DIR/hyperconverged.yaml
```

Comment:
This line assumes the hyperconverged custom resource will always be named `hyperconverged`? Is that the case?